### ISSUE TITLE
Error saving checkpoint

### ISSUE THREAD

**Issue by rhamnett** [jdb78/pytorch-forecasting#49]
Saving latest checkpoint..
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-14-d89dc578d7fe> in <module>
      1 # fit network
      2 trainer.fit(
----> 3     tft, train_dataloader=train_dataloader, val_dataloaders=val_dataloader,
      4 )

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/trainer/states.py in wrapped_fn(self, *args, **kwargs)
     46             if entering is not None:
     47                 self.state = entering
---> 48             result = fn(self, *args, **kwargs)
     49 
     50             # The INTERRUPTED state can be set inside the run function. To indicate that run was interrupted

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py in fit(self, model, train_dataloader, val_dataloaders, datamodule)
   1082             self.accelerator_backend = CPUBackend(self)
   1083             self.accelerator_backend.setup(model)
-> 1084             results = self.accelerator_backend.train(model)
   1085 
   1086         # on fit end callback

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/accelerators/cpu_backend.py in train(self, model)
     37 
     38     def train(self, model):
---> 39         results = self.trainer.run_pretrain_routine(model)
     40         return results

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py in run_pretrain_routine(self, model)
   1237 
   1238         # CORE TRAINING LOOP
-> 1239         self.train()
   1240 
   1241     def _run_sanity_check(self, ref_model, model):

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/trainer/training_loop.py in train(self)
    407                 if self.should_stop:
    408                     if (met_min_epochs and met_min_steps):
--> 409                         self.run_training_teardown()
    410                         return
    411                     else:

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_lightning/trainer/training_loop.py in run_training_teardown(self)
   1143             # model hooks
   1144             if self.is_function_implemented('on_train_end'):
-> 1145                 self.get_model().on_train_end()
   1146 
   1147         if self.logger is not None:

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_forecasting/models/temporal_fusion_transformer/__init__.py in on_train_end(self)
    583     def on_train_end(self):
    584         if self.log_interval(train=True) > 0:
--> 585             self._log_embeddings()
    586 
    587     def step(self, x, y, batch_idx, label="train"):

~/anaconda3/envs/TF2/lib/python3.7/site-packages/pytorch_forecasting/models/temporal_fusion_transformer/__init__.py in _log_embeddings(self)
    868             labels = self.hparams.embedding_labels[name]
    869             self.logger.experiment.add_embedding(
--> 870                 emb.weight.data.cpu(), metadata=labels, tag=name, global_step=self.global_step
    871             )

~/anaconda3/envs/TF2/lib/python3.7/site-packages/torch/utils/tensorboard/writer.py in add_embedding(self, mat, metadata, label_img, global_step, tag, metadata_header)
    786         save_path = os.path.join(self._get_file_writer().get_logdir(), subdir)
    787 
--> 788         fs = tf.io.gfile.get_filesystem(save_path)
    789         if fs.exists(save_path):
    790             if fs.isdir(save_path):

AttributeError: module 'tensorflow._api.v2.io.gfile' has no attribute 'get_filesystem'
```

**Comment by rhamnett** [comment]
https://github.com/pytorch/tutorials/issues/762

**Comment by jdb78** [comment]
This is a known problem with pytorch (https://github.com/pytorch/pytorch/issues/30966). To remedy it, uninstall tensorflow and then reinstall tensorboard.

**Comment by rhamnett** [comment]
@jdb78 thanks for the reply, for the moment I have successfully used the fix at the bottom of the link in my second comment. Thanks for creating this library and keep up the great work!

```
import tensorflow as tf
import tensorboard as tb
tf.io.gfile = tb.compat.tensorflow_stub.io.gfile
```

**Comment by jdb78** [comment]
Mentioned your fix in PR #54. Thanks!!
